cn/docs/tutorials/object-management-kubectl/object-management.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Kubernetes 对象管理
+redirect_from:
+- "/docs/concepts/tools/kubectl/object-management-overview/"
+- "/docs/concepts/tools/kubectl/object-management-overview.html"
+- "/docs/user-guide/working-with-resources/"
+- "/docs/user-guide/working-with-resources.html"
+---
+
+{% capture overview %}
+`kubectl` 命令行工具支持 Kubernetes 对象几种不同的创建和管理方法。本文档概述了不同的方法.
+{% endcapture %}
+
+{% capture body %}
+
+## 管理技巧
+
+**警告:** Kubernetes 对象应该只使用一种技术进行管理。混合使用不同的技术，会导致相同对象出现未定义的行为。
+
+| 管理技术             | 操作         |推荐环境 | 支持撰写  | 学习曲线 |
+|----------------------------------|----------------------|------------------------|--------------------|----------------|
+| 命令式的方式              | 活动对象         | 开发项目   | 1+                 | 最低         |
+| 命令式对象配置  | 单文件     | 生产项目    | 1                  | 中等       |
+| 声明式对象配置 | 文件目录 | 生产项目    | 1+                 | 最高        |
+
+## 命令式的方式
```

Comment:
@xingzhou Command itself contains the meaning of the composition of the order, so there is no such translation.